src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function get-margin-size($number-of-day) {
+  @return (($square-size * $number-of-day) + ($gap-size * $number-of-day));
+}
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)